### PR TITLE
[cli] Update documentation

### PR DESCRIPTION
**Pull request by gregnazario**
### Description
Start of updating CLI documentation

### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4981)

### CODE REVIEW

**Review by rajkaramchedu — APPROVED** [review]

File: crates/aptos/src/governance/mod.rs [hunk 1]
```diff
@@ -440,7 +450,7 @@ impl CliCommand<Vec<TransactionSummary>> for SubmitVote {
             (false, true) => ("No", false),
             (_, _) => {
                 return Err(CliError::CommandArgumentError(
-                    "Must choose either --yes or --no".to_string(),
+                    "Must choose only either --yes or --no".to_string(),
```
Comment (clay-aptos) [inline crates/aptos/src/governance/mod.rs:453]: I find the "only" here superfluous but your call.

**Review by clay-aptos — APPROVED** [review]
Excellent additions, Greg. I am a little confused why you added "Command" as the lead in some comments and removed it from others but assume that it's just my naivety.  Thank you very much for improving our knowledge set.

**Review by clay-aptos — APPROVED** [review]

**Comment by gregnazario** [comment]
> Excellent additions, Greg. I am a little confused why you added "Command" as the lead in some comments and removed it from others but assume that it's just my naivety.  Thank you very much for improving our knowledge set.

It's probably because I've been writing it for a few days, but I keep getting distracted.  I'm removing it on all of them soon

**Comment by clay-aptos** [comment]
> 

Well LGTM and appreciated.

**Review by movekevin — APPROVED** [review]

**Pull request merged**